test/builtin-functions.ll

```diff
@@ -16,7 +16,7 @@
 ; CHECK-FUNCTION-FORMAT-REV: declare spir_func i64 @_Z26__spirv_BuiltInWorkgroupIdi(i32)
 ; CHECK-FUNCTION-FORMAT-OCL-REV: declare spir_func i64 @_Z12get_group_idj(i32)
 
-; CHECK-GLOBAL-FORMAT-REV: @__spirv_BuiltInWorkgroupId = external addrspace(7) constant <3 x i64>
```

Comment:
I think this was wrongly being set to addrspace(7) by the reverse translator. Adding @sarnex for comments.

Thanks

Reply:
I'm not sure but the addrspace is not the point of the test so you can change the addrspace as you see fit.